app/mailers/peer_to_peer_match_mailer.rb

```diff
@@ -0,0 +1,13 @@
+class PeerToPeerMatchMailer < ApplicationMailer
+  def peer_to_peer_email(contribution, peer_to_peer_match_params)
```

Comment:
Minor: the second arg could be expanded into keyword args:
```suggestion
  def peer_to_peer_email(contribution, peer_alias:, message:)
```

Reply:
@exbinary, Thanks for the tip. There was no point in passing all the params here. Made necessary changes here.